### PR TITLE
fix: absolute url http/https

### PR DESCRIPTION
**Pull request by theogainey**
### Description
fix absolute url to use http for development env and https for everything else

### Spec

See Story: [213]((https://sparkbox.atlassian.net/browse/FSA22V1-213))

### Validation

* [ ] This PR has code changes, and our linters still pass.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all tests pass.
4. Confirm that [id].jsx works in both local development and in production preview deploy

### CODE REVIEW

File: pages/[id].jsx [hunk 1]
```diff
@@ -31,7 +31,8 @@ export default function DetailPage({ fetchedTeamMember }) {
 export async function getServerSideProps({ params, req }) {
   const { id } = params;
   const { host } = req.headers;
-  const baseUrl = `http://${host}/api/fellowship/`;
+  const scheme = process.env.NODE_ENV === 'development' ? 'http://' : 'https://';
+  const baseUrl = `${scheme}${host}/api/fellowship/`;
```
Comment (theogainey) [inline pages/[id].jsx:35]: This isn't exactly the fix the issue described, but exporting the environment through `config/index.js` was not working as expected and NextJS ships with support for environment variables so this seemed like a good way to go to me. As always if there is something I am missing I am happy to make the changes.

Comment (dustin-jw) [reply pages/[id].jsx:35]: Okay, this works for now. I would like to see a `.env.example`, though that we can add to as we need environment variables.

```
# set to 'development' for local, anything else will be considered production
NODE_ENV=development
```

Comment (theogainey) [reply pages/[id].jsx:35]: Sounds good! I pushed the addition

**Review by dustin-jw — APPROVED** [review]
Nicely done! @arnest00 I'll hand it over to you!

**Review by arnest00 — APPROVED** [review]

**Pull request merged**